### ISSUE TITLE
Cannot build it on Linuxmint 20.04

### ISSUE THREAD

**Issue by ikarlmarx** [neutrinolabs/pulseaudio-module-xrdp#75]
log:
No such script: /usr/share/debootstrap/scripts/uma

Anybody can help?
Thank you very much.

**Comment by matt335672** [comment]
Hi @ikarlmarx 

Thanks for raising this.

I've managed to reproduce it. It seems Mint is built on top of Ubuntu 'focal', but actually changes the codename of the system as reported by the `lsb_release -cs` command to `una` from `focal`.

I've added a branch to my development repository to support this. Please try the following in your git repository, if you're happy to work from my repo:-

```
git remote add matt335672 https://github.com/matt335672/pulseaudio-module-xrdp.git
git fetch matt335672
git checkout -b suite_support --track matt335672/suite_support
```

Then use this command to try to build the module:-

```
scripts/install_pulseaudio_sources_apt_wrapper.sh --suite=focal
```

Let me know how it goes

**Comment by matt335672** [comment]
@ikarlmarx 

Have you had a chance to look at this yet?

Thanks.

**Comment by ikarlmarx** [comment]
git remote add matt335672 https://github.com/matt335672/pulseaudio-module-xrdp.git:

fatal: not git lib（or any parent dir）：.git

**Comment by ikarlmarx** [comment]
Sorry I just saw it.

**Comment by matt335672** [comment]
Sorry - I'm assuming you're building from git, and it doesn't look like you are.

Try this:-

```
git clone https://github.com/matt335672/pulseaudio-module-xrdp
cd pulseaudio-module-xrdp
git checkout suite_support
scripts/install_pulseaudio_sources_apt_wrapper.sh --suite=focal
```

then follow the rest of the build instructions.

**Comment by ikarlmarx** [comment]
> Sorry - I'm assuming you're building from git, and it doesn't look like you are.
> 
> Try this:-
> 
> ```
> git clone https://github.com/matt335672/pulseaudio-module-xrdp
> cd pulseaudio-module-xrdp
> git checkout suite_support
> scripts/install_pulseaudio_sources_apt_wrapper.sh --suite=focal
> ```
> 
> then follow the rest of the build instructions.

Sorry for a long time no reply.

Everything is good, except when I ran:

scripts/install_pulseaudio_sources_apt_wrapper.sh --suite=focal

I Got:

** Unsupported suite 'focal'

I checked the system(Linux Mint 20.3). And there is no debootstrap.

**Comment by ikarlmarx** [comment]
It is DONE.
I installed debootstrap, and some other tools such as autoconf, automake, libtool.
Then:
./bootstrap && ./configure PULSE_DIR=~/pulseaudio.src
make
sudo make install

and then:
 ls $(pkg-config --variable=modlibexecdir libpulse) | grep xrdp
They are there.

Reboot, done.

Thank you very much.

**Comment by matt335672** [comment]
Great - thanks for the feedback. I'll get the script and the Wiki updated.

**Comment by matt335672** [comment]
Closed by #78
